src/rgx/block.rgx

```diff
@@ -12,7 +12,6 @@
 /wp-(app|admin|config|contents?|includes|login|rss.*|register)
 /zimbra
 %25(2[26]|3([aA]|[dD])|5[cC]|7[cC])
-%25[^2-7a-fA-F][^0-9a-fA-F]
```

Comment:
This expression is to block requests with '%' not followed by two valid hexadecimal characters, as this again provokes a 500 and message `Illegal query: Invalid input 'g', expected HEXDIG (line 1, column 313)`.

Can we add an example to the block list for this instead?

Reply:
Noticed that we actually need two expressions:
```
%25[^2-7a-fA-F]             # match % followed by anything but sensible hexadecimal
%25[2-7a-fA-F][^0-9a-fA-F]  # match % followed by sensible hexadecimal but not another hexadecimal
```

Reply:
I think we can simplify the regexp to only allow lowercase letters, numbers, `-` and `_`.
at least that's what we discussed with @davidnuescheler.

so we don't need this super complex regexp anymore.

Reply:
hmm... paths like 

`/foo/bar%1`
`/foo/bar%`

aren't blocked but still trigger `500`s

Reply:
if we only allow letters, digits and dashes, `/foo/bar%1` would be blocked.

Reply:
> I think we can simplify the regexp to only allow lowercase letters, numbers, - and _.
at least that's what we discussed with @davidnuescheler.

Didn't we had cases just recently where the path did contain `%20` (live site)?

Reply:
:-) user error....

ok, allow lowercase letters, -, _, ., and `%20`. but that's it.

Reply:
> so we don't need this super complex regexp anymore.

which one? 😄

Reply:
> I think we can simplify the regexp to only allow lowercase letters, numbers, `-` and `_`

no extension?

Reply:
> I think we can simplify the regexp to only allow lowercase letters, numbers, - and _

If this relates to **all** request URLs for theblog, we already have at least 4 documents that won't work, because their URL contains %-encoded characters.

Reply:
@dominique-pfister please add the examples directly to the branch

Reply:
Done, these were taken from https://github.com/adobe/theblog/issues/260